src/App/component/ProjectAvatar.jsx

```diff
@@ -79,9 +90,53 @@ function ProjectAvatar(props) {
     setAddRepoDialogOpen(true)
   }
 
+  const openDeletionAlert = () => {
+    setDeletionAlert(true)
+  }
+
+  const closeDeletionAlert = () => {
+    setDeletionAlert(false)
+  }
+
+  const deleteProject = () => {
+    Axios.post(`http://localhost:9100/pvs-api/project/remove/${props.project.projectId}`, "",
+      {headers: {"Authorization": `${jwtToken}`}})
+      .then(() => {
+        closeDeletionAlert()
+        props.reloadProjects()
+      })
+      .catch((e) => {
+        alert(e.response.status)
+        console.error(e)
+      })
+  }
+
   return (
     <div>
       <Box className={props.size === 'large' ? classes.large : classes.small}>
+      {props.size === 'large' &&
+      <Button onClick={openDeletionAlert}>X</Button>
+      }
+      <Dialog
+      open={deletionAlert}
+      onClose={closeDeletionAlert}
+      aria-labelledby="alert-dialog-title"
+      aria-describedby="alert-dialog-description">
+        <DialogTitle id="alert-dialog-title">
+          {"Are You Sure You Want to Delete This Project?"}
+        </DialogTitle>
+        <DialogContent>
+          <DialogContentText id="alert-dialog-description">
+            If you delete the project, you cannot restore it.
```

Comment:
```suggestion
            You can restore it after delete.
```

Reply:
We don't plan to make a recycle bin. So in user perspective, there is no way to restore their abandoned projects.